.evergreen/publish.sh

```diff
@@ -7,22 +7,28 @@ set -o errexit  # Exit the script with error if any of the commands fail
 ############################################
 #            Main Program                  #
 ############################################
+RELEASE=${RELEASE:false}
 
-echo ${RING_FILE_GPG_BASE64} | base64 -d > ${PROJECT_DIRECTORY}/secring.gpg
-
-trap "rm ${PROJECT_DIRECTORY}/secring.gpg; exit" EXIT HUP
+echo ${RING_FILE_GPG_BASE64} | base64 -d > /tmp/secring.gpg
```

Comment:
Moved to /tmp so it doesn't make the working directory dirty

Reply:
As the trap is designed to clean it up on exit of this shell script does it make the working directory dirty? I'm not sure what that means / the ramifications of that are.  I'd rather the file be kept local than spread outside to a potentially globally accessible location.

Reply:
Reverted this change and instead added `*.gpg` to `.gitignore`